packages/mds-agency/tests/integration-tests.ts

```diff
@@ -1346,6 +1348,43 @@ describe('Tests API', () => {
       })
   })
 
+  it('verifies get device defaults to `deregister` if cache misses reads for associated events', async () => {
+    await request
+      .post('/vehicles')
+      .set('Authorization', AUTH)
+      .send(JUMP_TEST_DEVICE_1)
+      .expect(201)
+
+    await request
+      .post(`/vehicles/${JUMP_TEST_DEVICE_1_ID}/event`)
+      .set('Authorization', AUTH)
+      .send({ device_id: JUMP_TEST_DEVICE_1, timestamp: now(), event_type: VEHICLE_EVENTS.deregister })
+      .expect(201)
+
+    // @ts-ignore: Spoofing garbage data
+    const result = await request
+      .get(`/vehicles/${JUMP_TEST_DEVICE_1_ID}`)
+      .set('Authorization', AUTH)
+      .expect(200)
+    test.assert(result.body.status === VEHICLE_STATUSES.inactive)
+    test.assert(result.body.prev_event === VEHICLE_EVENTS.deregister)
+  })
+
+  it('verifies get multiple devices defaults to `inactive` if event_type is missing', async () => {
```

Comment:
Test name isn't really accurate

Reply:
I did push an updated name, but as we've discussed, I have a pretty vague idea of what's going on in agency and it'd be helpful for you to suggest a better test name if the new one I suggested isn't up to snuff.